frontend/src/Functions/components.jsx

```
@@ -18,22 +18,21 @@ export function MedalIcon({ rank }) {
     const text = part.slice(1);
-    const color = COD_2_COLORS[colorNumber];
+    const color = COD2_COLORS[colorNumber];
 
     return (
       <span className={color} key={index}>
         {text}
       </span>
     );
   });
+  
+  return elements.length === 1 ? elements[0] : elements;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)